fix: docstring for imagetorch encoder (#4422)
jina-ai/jina-hub: encoders/image/ImageTorchEncoder/README.md

# ImageTorchEncoder

> `ImageTorchEncoder` encodes data from a ndarray, potentially BatchSize x (Channel x Height x Width) into a ndarray of `BatchSize * d`. Internally, :class:`ImageTorchEncoder` wraps the models from `tensorflow.keras.applications`. https://keras.io/applications/

Reviewer: can you fix this in the attached image: 
![Uploading image.png…]()

## Usage:

Initialise this Executor specifying parameters i.e.:

| `param_name`  | `param_remarks` |
| ------------- | ------------- |
| `model_path`  | the directory path of the model in the `SavedModel` format  |
| `model_name`  | includes `resnet18`, `alexnet`, `squeezenet1_0`, `vgg16`, `densenet161`, `inception_v3` |
| `channel_axis`| axis of the color channel  |
| `pool_strategy` | strategy of pooling operation |

> The model is pre-trained on [MobileNetV2] data

Reviewer: missing link for `MobileNetV2`

Reviewer 2: MobileNetV2 data makes no sense, the dataset is ImageNet. But anyway I think is not worth, I would say `model` is one of `resnet18`, `alexnet`, `squeezenet1_0`, `vgg16`, `densenet161`, `inception_v3`

Author: I've updated the doc to reflect the right models.

The pretrained default path is the result of downloading the models in `download.sh`

### Snippets:

Initialise ImageTorchEncoder:

`ImageTorchEncoder(model_path='pretrained', channel_axis=1, metas=metas, model_name=MobileNetV2)`

Users can use Pod images in several ways:

- Run with Docker (`docker run`)
  - ```bash

>     docker run jinahub/pod.encoder.imagetorchencoder:0.0.8-0.9.28 --port-in 55555 --port-out 55556

Reviewer 3: ```suggestion
    docker run jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4 --port-in 55555 --port-out 55556
```

    ```

- Flow API
  - ```python
    from jina.flow import Flow
    f = (Flow()

>         .add(name='my_encoder', uses='jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4', port_in=55555, port_out=55556)

Reviewer 4: ```suggestion
        .add(name='my_encoder', uses='docker://jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4', port_in=55555, port_out=55556)
```

>         .add(name='my-encoder', image='jinahub/pod.encoder.imagetorchencoder:0.0.8-0.9.28', port_in=55555, port_out=55556)

Reviewer 3: ```suggestion
        .add(name='my_encoder', uses='jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4', port_in=55555, port_out=55556)
```

    ```

- Jina CLI
  - ```bash

>     jina pod --uses jinahub/pod.encoder.imagetorchencoder:0.0.8-0.9.28 --port-in 55555 --port-out 55556

Reviewer 3: ```suggestion
    jina pod --uses docker://jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4 --port-in 55555 --port-out 55556
```

    ```

- Conventional local usage with `uses` argument
  - ```bash
    jina pod --uses hub/example/config.yml --port-in 55555 --port-out 55556
    ```

- Docker command

>   - Specify the image name along with the version tag. The snippet below uses Jina version `0.9.20`

Reviewer 3: Or make sure the version is consistent
```suggestion
  - Specify the image name along with the version tag. The snippet below uses Jina version `1.0.4`
```

  - ```bash

>     docker pull jinahub/pod.encoder.imagetorchencoder:0.0.8-0.9.28

Reviewer 3: ```suggestion
    docker pull jinahub/pod.encoder.imagetorchencoder:0.0.8-1.0.4
```
